pyproject.toml

```diff
@@ -17,7 +17,7 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.8"
```

Comment:
Why are you changing the Python version here for the install requires? I don't see any use of features that are >= Python3.8, and Nautobot itself still supports Python 3.7.

Reply:
There was discussion of making 3.8 the default due to 3.7 being EOL soon and not wanting to have to go through the whole ordeal again. If we want to stick to 3.7 that's fine with me but it is EOL in less than a year ([27 Jun 2023](https://endoflife.date/python)).

Reply:
There is still an open discussion about removing Python 3.7, but since Nautobot Core has not yet removed support, this change in the PR would cause anyone using Python3.7 for their Nautobot environment to be unable to install this plugin.

Reply:
I still firmly believe we should be going to 3.8 with 3.7 being EOL in less than a year but we can address it later I guess. I'm pushing changes to revert this bit of the PR though.